src/Yoti.Auth/DigitalIdentity/ShareSessionRequestBuilder.cs

```diff
@@ -1,6 +1,7 @@
 ﻿using System.Collections.Generic;
 using Yoti.Auth.DigitalIdentity.Extensions;
 using Yoti.Auth.DigitalIdentity.Policy;
+using Yoti.Auth.ShareUrl.Policy;
```

Comment:
Why was this import added - `using Yoti.Auth.ShareUrl.Policy`? It doesn't seem to have any use here.

Reply:
removed